pkg/pipeline/message.go

```diff
@@ -21,62 +21,52 @@ type MessageType int
 const (
 	// MessageTypeUnknown is unknown message type
 	MessageTypeUnknown MessageType = iota
-	// MessageTypeLifecycle is lifecycle message type
-	MessageTypeLifecycle
-	// MessageTypeRowChangedEvent is the row changed event message type
-	MessageTypeRowChangedEvent
+	// MessageTypeCommand is command message type
+	MessageTypeCommand
+	// MessageTypePolymorphicEvent is the row changed event message type
+	MessageTypePolymorphicEvent
 )
 
 // Message is a vehicle for transferring information between nodes
 type Message struct {
 	// TODO add more kind of messages
 	// Tp is the type of Message
 	Tp MessageType
-	// Lifecycle represents the message about the lifecycle
-	Lifecycle *LifecycleMessage
-	// RowChangedEvent represents the row change event
-	RowChangedEvent *model.RowChangedEvent
+	// Command is the command in this message
+	Command *Command
+	// PolymorphicEvent represents the row change event
+	PolymorphicEvent *model.PolymorphicEvent
```

Comment:
I noticed `*model.PolymorphicEvent` type is used here, which may cause the library less universal. If we use an interface here, more assertation overhead may be introduced. Just curious about the design mode, not need to address.

Reply:
the pipeline framework is designed for processor mainly, so I didn't  using `interface{}` here

Reply:
If it is used for processor only, I prefer moving it to another package

Reply:
Where do you think better?

Reply:
some path like `github.com/pingcap/ticdc/cdc/xxx`

Reply:
Not a big problem